CMakeLists.txt

```diff
@@ -28,12 +23,7 @@ find_package(catkin REQUIRED
 ## LIBRARIES: libraries you create in this project that dependent projects also need
 ## CATKIN_DEPENDS: catkin_packages dependent projects also need
 ## DEPENDS: system dependencies of this project that dependent projects also need
-catkin_package(
-   INCLUDE_DIRS include
-   LIBRARIES ${PROJECT_NAME}_pass_through_filter_point_cloud2
-   CATKIN_DEPENDS roscpp pluginlib filters tf2_ros pcl_ros pcl_conversions sensor_msgs
-   #DEPENDS PCL
-)
+catkin_package()
```

Comment:
It is not expected that any other package will link to the filter provided by this package. In such case, `catkin_package()` should be empty because all the information within it are just for downstream packages trying to link to this one.